kolena/_experimental/classification/utils.py

```diff
@@ -251,3 +254,75 @@ class is `None`.
             curves=curves,
         )
     return None
+
+
+def compute_threshold_curves(
+    ground_truths: List[Optional[ClassificationLabel]],
+    inferences: List[ScoredClassificationLabel],
```

Comment:
Is it expected that ground truths and inferences correspond to the same sample (for the same given index)? If so, should probably be included in the documentation.